test/test_transforms_v2.py

```diff
@@ -1935,7 +1935,13 @@ def test_detection_preset(image_type, data_augmentation, to_tensor, sanitize):
 @pytest.mark.parametrize(
     "labels_getter", ("default", "labels", lambda inputs: inputs["labels"], None, lambda inputs: None)
 )
-def test_sanitize_bounding_boxes(min_size, labels_getter):
+@pytest.mark.parametrize("sample_type", (tuple, dict))
+def test_sanitize_bounding_boxes(min_size, labels_getter, sample_type):
+
+    if sample_type is tuple and not isinstance(labels_getter, str):
+        # The labels_getter callable used in this test don't work if the input is a tuple.
+        return
```

Comment:
Technically, the only one it doesn't work with is  `lambda inputs: inputs["labels"]`, but oh well

Reply:
Would you mind putting that in the code comment?